File: php/classes/Aksesoris.php
<?php

require_once('Petshop.php');

class Aksesoris extends Petshop {
    protected $jenis;
    protected $bahan;
    protected $warna;

    public function __construct($jenis = "", $bahan = "", $warna = "") {
        parent::__construct();
        $this->jenis = $jenis;
        $this->bahan = $bahan;
        $this->warna = $warna;
    }

    // Getter Methods
    public function getJenis() {
        return $this->jenis;
    }

    public function getBahan() {
        return $this->bahan;
    }

    public function getWarna() {
        return $this->warna;
    }

    // Setter Methods
    public function setJenis($jenis) {
        $this->jenis = $jenis;
        return $this;
    }

    public function setBahan($bahan) {
        $this->bahan = $bahan;
        return $this;
    }

    public function setWarna($warna) {
        $this->warna = $warna;
        return $this;
    }
}
?>

File: cpp/Aksesoris.cpp
#include <iostream>
#include <string>
#include <list>
#include "Petshop.cpp"

using namespace std;

class Aksesoris: public Petshop
{
    private:
        string jenis;
        string bahan;
        string warna;
    public:
        Aksesoris()
        {
            this->jenis = "";
            this->bahan = "";
            this->warna = "";
        }

        string get_jenis() const// membaca nama
        {
            return this->jenis;
        }
        
        void set_jenis(string jenis) // mengubah nama
        {
            this->jenis = jenis;
        }

        string get_bahan() const// membaca bahan
        {
            return this->bahan;
        }
        
        void set_bahan(string bahan) // mengubah bahan
        {
            this->bahan = bahan;
        }

        string get_warna() const// membaca warna
        {
            return this->warna;
        }
        
        void set_warna(string warna) // mengubah warna
        {
            this->warna = warna;
        }

        ~Aksesoris()
        {
            
        }
};